src/ca65/instr.c

```diff
@@ -783,7 +788,7 @@ static const InsTable* InsTabs[CPU_COUNT] = {
     (const InsTable*) &InsTab65816,
     (const InsTable*) &InsTabSweet16,
     (const InsTable*) &InsTabHuC6280,
-    0,                                  /* Mitsubishi 740 */
+    0,
```

Comment:
I stand correct, Uz looked into adding the Mitsubishi/Renesa 740 in 2003: http://www.cc65.org/mailarchive/2003-08/3202.html